ca/ca_test.go

```diff
@@ -193,7 +193,7 @@ func (m *mockSA) SerialExists(ctx context.Context, req *sapb.Serial) (*sapb.Exis
 }
 
 func (m *mockSA) GetCertificate(ctx context.Context, serial string) (core.Certificate, error) {
-	return core.Certificate{}, berrors.NotFoundError("cannae find tha cert")
+	return core.Certificate{}, berrors.NotFoundError("cannot find the cert")
```

Comment:
I think this was a case of someone having some fun with a mock error message. I "fixed" it anyway, sorry for killing the fun :-)

Reply:
This is Scottish erasure.

Reply:
![damn scots!](https://media.giphy.com/media/v2Mkppa5Lrspa/giphy.gif)